File: cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main (void)
{
    float dollars;
// propt user for input of change owed while input is number higher of 0
    do 
    {
    dollars =get_float("change owed ?");
    }

    while (dollars <0);
// input of dollars will be multiplied by 100 and rounded, in order to get cents     
    int cents;
    cents = round(dollars *100);


    int coins = 0;
    while (cents >=25)
    {
    cents -=25;
    coins++;
    }

 while (cents >=10)
    {
    cents -=10;
    coins++;
    }

 while (cents >=5)
    {
    cents -=5;
    coins++;
    }

    coins +=cents;
 


    printf("%i",coins);

    }

File: cash.py
from cs50 import get_float

# propt user for input of change owed while input is number higher of 0

while True:
    dollars = get_float("change owed : ")
    if dollars >0:
        break
#input of dollars will be multiplied by 100 and rounded, in order to get cents 
coins = 0

cents = 0

cents = round(dollars * 100)

while True:
    if (cents >= 25):
        cents -= 25
        coins += 1
    else:
        break

while cents < 25: 
        if (cents >= 10):
            cents -= 10
            coins += 1
        else:
            break

            
while  cents <10: 
    if (cents >= 5):
        cents -= 5
        coins += 1
    else:
        break
        
coins += cents
        
print(coins)
    

